### PR TITLE
Clarify PHP version support and fix format errors

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

Fix a formatting issue in PHP version constraint information for URL rewrites module.
- The | character in the version constraints caused issues rendering the callout.
-  Changed PHP version support from annotation to English

## Affected DevDocs pages

- https://devdocs.magento.com/cloud/configure/import-url-rewrites.html

### CODE REVIEW

File: src/cloud/configure/import-url-rewrites.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ functional_areas:
 You can easily migrate to the {{site.data.var.ece}} platform without losing SEO rankings and traffic. Use the `magento/url-rewrite-import-export` module to redirect traffic from your old, indexed URLs to new URLs.
 
 {:.bs-callout-info}
-This module supports ~7.0.13|~7.1.0|~7.2.0|~7.3.0 PHP versions and is available for Magento version 2.2.x and 2.3.x only.
+This module supports the following PHP versions 7.0.13 and later patch releases and all 7.1.0 and 7.2.0 patch releases. The module is available for Magento version 2.2.x and 2.3.x only.
```
Comment (meker12) [inline src/cloud/configure/import-url-rewrites.md:13]:
```suggestion
This module supports the following PHP versions 7.0.13 and later patch releases and all 7.1.x and 7.2.x patch releases. The module is available for Magento version 2.2.x and 2.3.x only.
```

Comment (shrielenee) [inline src/cloud/configure/import-url-rewrites.md]: Could this be better by removing "the following"?

File: src/cloud/configure/import-url-rewrites.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ functional_areas:
 You can easily migrate to the {{site.data.var.ece}} platform without losing SEO rankings and traffic. Use the `magento/url-rewrite-import-export` module to redirect traffic from your old, indexed URLs to new URLs.
 
 {:.bs-callout-info}
-This module supports ~7.0.13|~7.1.0|~7.2.0|~7.3.0 PHP versions and is available for Magento version 2.2.x and 2.3.x only.
+This module supports the following PHP versions 7.0.13 and later patch releases and all 7.1.x and 7.2.x patch releases. The module is available for Magento version 2.2.x and 2.3.x only.
```
Comment (meker12) [inline src/cloud/configure/import-url-rewrites.md:13]:
```suggestion
This module supports PHP versions 7.0.13 and later patch releases and all 7.1.x and 7.2.x patch releases. The module is available for Magento version 2.2.x and 2.3.x only.
```

**Comment by meker12** [comment]
running tests

**Pull request merged**